cluster/service/cluster.go

```diff
@@ -38,25 +32,25 @@ func NewClusterService(context servicectx.ServiceContext, loader ConfigLoader) s
 	}
 }
 
-// CreateOrSaveOSOClusterFromConfig creates clusters or save updated cluster info from config
-func (c clusterService) CreateOrSaveOSOClusterFromConfig(ctx context.Context) error {
-	for _, osoCluster := range c.loader.GetOSOClusters() {
+// CreateOrSaveClusterFromConfig creates clusters or save updated cluster info from config
+func (c clusterService) CreateOrSaveClusterFromConfig(ctx context.Context) error {
+	for _, configCluster := range c.loader.GetClusters() {
 		cluster := &repository.Cluster{
```

Comment:
you may want to avoid using `cluster` as a variable name to avoid colliding with an existing pkg

Reply:
addressed in ec3061b